examples/pnpm/.sauce/config.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1alpha
+kind: playwright
+sauce:
+  region: us-west-1
+  concurrency: 10 # Controls how many suites are executed at the same time.
+  metadata:
+    tags:
+      - e2e
+      - release team
+      - other tag
+playwright:
+  version: 1.37.1 # See https://docs.saucelabs.com/web-apps/automated-testing/playwright/#supported-testing-platforms for a list of supported versions.
+  configFile: playwright.config.js # See https://docs.saucelabs.com/web-apps/automated-testing/playwright/yaml/#configfile for a list of supported configuration files.
+# Controls what files are available in the context of a test run (unless explicitly excluded by .sauceignore).
+rootDir: ./
+suites:
+  - name: "Chromium"
+    platformName: "Windows 11"
+    #platformName: "macOS 12"
+    testMatch: ['.*.js']
+    preExec:
+      - npx pnpm install
+    params:
+      browserName: "chromium"
+      project: "chromium"
+npm:
+  packages:
+    lodash: "*"
```

Comment:
No dependencies should be declared here and all dependencies are managed by pnpm.